apps/pie-docs/src/components/chip/chip.md

```diff
@@ -0,0 +1,70 @@
+---
+eleventyNavigation:
+    key: 'Chip'
+    parent: Components
+---
+
+## Overview
+
+The purpose of chips is to provide a visual representation of a specific entity or attribute, such as a selected option or a labelled category. Chips can also be interactive, allowing users to remove or modify the selected choices.
+
+Chips are commonly used in various contexts, including filtering options, search results or any situation where concise and visually distinct information needs to be displayed.
+
+
+{% contentPageImage {
+    src:"../../assets/img/components/chip/overview.svg",
+    alt: "A group of chips that are horizontally stacked together."
```

Comment:
Nit: "horizontally stacked" is a bit of a contradiction. Maybe "displayed as a single row" or something?

YCFTL (you can fix this later)